plutus-core/plutus-core/src/Language/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -57,19 +57,15 @@ type WithMemory f (uni :: GHC.Type -> GHC.Type) (fun :: GHC.Type) = f TyName Nam
 -- | Counts size in machine words (64bit for the near future)
 newtype ExMemory = ExMemory Integer
   deriving (Eq, Ord, Show)
-  deriving newtype (Num, NFData)
+  deriving newtype (Num, Pretty, NFData)
   deriving (Semigroup, Monoid) via (Sum Integer)
--- You have to use a standalone declaration for deriving a 'PrettyBy' instance.
--- Yes, I also hate that.
-deriving newtype instance PrettyDefaultBy config Integer => PrettyBy config ExMemory
 
 -- TODO: 'Integer's are not particularly fast. Should we use @Int64@?
 -- | Counts CPU units - no fixed base, proportional.
 newtype ExCPU = ExCPU Integer
   deriving (Eq, Ord, Show)
-  deriving newtype (Num, NFData)
+  deriving newtype (Num, Pretty, NFData)
   deriving (Semigroup, Monoid) via (Sum Integer)
-deriving newtype instance PrettyDefaultBy config Integer => PrettyBy config ExCPU
```

Comment:
We don't need to use configurable pretty-printing for things that only get pretty-printed in a single way, so I replaced `PrettyBy config` instances with `Pretty` instances for costing-related stuff.

Reply:
> We don't need to use configurable pretty-printing for things that only get pretty-printed in a single way

Apparently we do, 'cause sometimes we pretty-print tuples where one component is allowed to be pretty-printed in a configurable manner and the other is not, so the whole things requires configurable pretty-printing, including the other component. So I returned `PrettyBy` instances.